lib/node_modules/@stdlib/ndarray/base/mskfilter/lib/main.js

```diff
@@ -0,0 +1,180 @@
+* var ybuf = new Float64Array( [ 1.0, 2.0, 3.0, 4.0 ] );
+*
+* // Define the shape of the input and mask arrays:
+* var shape = [ 3, 1, 2 ];
+*
+* // Define the array strides:
+* var s = [ 4, 4, 1 ];
+*
+* // Define the index offsets:
+* var ox = 0;
+* var om = 0;
+*
+* // Create the input, mask and output ndarray-like objects:
+* var x = {
+*     'dtype': 'float64',
+*     'data': xbuf,
+*     'shape': shape,
+*     'strides': s,
+*     'offset': ox,
+*     'order': 'row-major'
+* };
+* var mask = {
+*     'dtype': 'float64',
+*     'data': mbuf,
+*     'shape': shape,
+*     'strides': s,
+*     'offset': om,
+*     'order': 'row-major'
+* };
+* var out = {
+*     'dtype': 'float64',
+*     'data': ybuf,
+*     'shape': [ 4 ],
+*     'strides': [ 1 ],
+*     'offset': 0,
+*     'order': 'row-major'
+* };
+*
+* // Copy elements:
+* mskfilter( [ x, mask, out ] );
+*
+* console.log( out.data );
+* // => <Float64Array>[ 1.0, 5.0, 9.0, 11.0 ]
+*/
+function mskfilter( arrays ) {
+    var ndims;
+    var mask;
+    var shm;
+    var shx;
+    var x;
+    var y;
+
+    // Unpack the ndarrays and standardize ndarray meta data:
+    x = ndarray2object( arrays[ 0 ] );
+    mask = ndarray2object( arrays[ 1 ] );
+    y = ndarray2object( arrays[ 2 ] );
+
+    // Verify that the input and mask arrays have the same number of dimensions...
+    shx = x.shape;
+    shm = mask.shape;
+    ndims = shx.length;
+    if ( ndims !== shm.length ) {
+        throw new Error( format( 'invalid arguments. Arrays must have the same number of dimensions (i.e., same rank). ndims(x) == %s. ndims(mask) == %s', ndims, shm.length ) );
+    }
+
+    // Verify that the input and mask arrays have the same shape...
+    for ( var i = 0; i < ndims; i++ ) {
+        if ( shx[ i ] !== shm[ i ] ) {
+            throw new Error( format( 'invalid arguments. Arrays must have the same shape. shape(x)[%s] == %s. shape(mask)[%s] == %s', i, shx[ i ], i, shm[ i ] ) );
+        }
+    }
+
+    // Verify that the output array has a one-dimensional shape...
+    if ( y.shape.length !== 1 ) {
+        throw new Error( 'invalid argument. Output array must have a one-dimensional shape. ndims(y) == %s', y.shape.length );
+    }
+
+    // Verify that the output array and input array have same data type...
+    if ( x.dtype !== y.dtype ) {
+        throw new Error( 'invalid arguments. Input and output arrays must have the same data type. dtype(x) == %s. dtype(y) == %s', x.dtype, y.dtype );
+    }
+
+    // Check whether we were provided an empty input ndarray...
+    if ( numel( shx ) === 0 ) {
+        return;
+    }
+
+    // Determine whether we can avoid iteration altogether...
+    if ( ndims == 0 ) {
+        if ( x.accessorProtocol || mask.accessorProtocol || y.accessorProtocol ) {
+            return ACCESSOR_MSKFILTER[ ndims ]( x, mask, y );
+        }
+        return MSKFILTER[ ndims ]( x, mask, y );
+    }
+}
+
+
+// EXPORTS //
+
+module.exports = mskfilter;
```

Comment:
@Jaysukh-409 Not clear to me why this should be the case, but can you fix all the indentation and missing trailing newlines. I would have expected that you had setup EditorConfig and your local dev env to prevent committing code such as this.